plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/rest/TestIcebergPolarisCatalogConnectorSmokeTest.java

```diff
@@ -147,6 +150,23 @@ protected void deleteDirectory(String location)
         }
     }
 
+    @Test
+    void testNestedNamespace()
+    {
+        String level1 = "level1_" + randomNameSuffix();
+        String level2 = "level2_" + randomNameSuffix();
+        assertUpdate("CREATE SCHEMA " + level1);
+        assertUpdate("CREATE SCHEMA \"" + level1 + "." + level2 + "\"");
+        assertThat(computeActual("SHOW SCHEMAS").getOnlyColumnAsSet())
+                .containsExactlyInAnyOrder(
```

Comment:
This assertion will start failing if other tests create a schema, right?

Reply:
ya that's true. Thanks for removing it